templates/ldap_custom_fragment.erb

```diff
@@ -0,0 +1,15 @@
+# pass everything that isn't a Basic auth request through to Pulp
+    SetEnvIfNoCase ^Authorization$ "Basic.*" USE_APACHE_AUTH=1
+    Order allow,deny
+    Allow from env=!USE_APACHE_AUTH
+    Satisfy Any
+
+    # configure basic auth
+    AuthType basic
+    AuthBasicProvider ldap
+    AuthName "Pulp"
+    AuthLDAPURL "<%= scope['pulp::ldap_url'] %>"
```

Comment:
It was possible to configure some lines with directories options, but not others.  custom_fragments are added at the end of the template.  Mixing custom_fragment with other options ended up quite messy.  At least this way, the output matches the [documented ](https://docs.pulpproject.org/user-guide/authentication.html#ldap-whole-api-example)way of configuring this feature.